engine/src/main/java/com/redhat/cloud/notifications/templates/Edge.java

```diff
@@ -0,0 +1,49 @@
+package com.redhat.cloud.notifications.templates;
+
+import com.redhat.cloud.notifications.models.EmailSubscriptionType;
+import io.quarkus.qute.CheckedTemplate;
+import io.quarkus.qute.TemplateInstance;
+
+// Name needs to be "Edge" to read templates from resources/templates/Edge
+public class Edge implements EmailTemplate {
+
+    @Override
+    public TemplateInstance getTitle(String eventType, EmailSubscriptionType type) {
+        if (eventType.equals("image-creation")) {
+            return Templates.imageCreationTitle();
+        }
+        return Templates.updateDeviceTitle();
```

Comment:
While this will work if you have two event types, it would be safer to make the following change so that `updateDeviceTitle` is not used by accident if a third event type is added in the future:
```suggestion
        if (eventType.equals("image-creation")) {
            return Templates.imageCreationTitle();
        } else if (eventType.equals("update-devices") {
            return Templates.updateDeviceTitle();
        }
        throw new UnsupportedOperationException(String.format(
                "No email title template for Edge event_type: %s and EmailSubscription: %s found.",
                eventType, type
        ));
```